eeip/eipclient.py

```diff
@@ -114,8 +114,13 @@ def unregister_session(self):
         self.__tcpClient_socket.send(bytearray(__encapsulation.to_bytes()))
         if self.__tcpClient_socket is not None:
             self.__stoplistening = True
-            self.__tcpClient_socket.shutdown(socket.SHUT_RDWR)
-            self.__tcpClient_socket.close()
+            try:
```

Comment:
Can you tell me your thoughts here? What failure mode are we eliminating and what possible new failure modes are we introducing?

Reply:
We're eliminating cases where this thread doesn't get cleaned up correctly due to exceptions. 

One drawback is that exceptions here won't get surfaced to the caller. I think that's ok - since this is in the close() path, those exceptions are not really actionable anyway.